.gitignore

```diff
@@ -0,0 +1,2 @@
+.history
```

Comment:
- [ ] As suggested by the previous reviewer, please also add the `node_modules` directory as stated by the project requirements.

Reply:
Added it. Thanks.